labgrid/remote/coordinator.py

```diff
@@ -960,6 +1043,25 @@ async def GetReservations(self, request: labgrid_coordinator_pb2.GetReservations
         reservations = [x.as_pb2() for x in self.reservations.values()]
         return labgrid_coordinator_pb2.GetReservationsResponse(reservations=reservations)
 
+    async def _release_session(self, client_session):
+        session = client_session.session
+
+        objs = self.session_manager.get_session_objs(session)
+        await self.lock.acquire()
+        for place in objs["places"]:
+            await self._release_place(place)
+        for reservation in objs["reservations"]:
+            self._release_reservation(reservation.token)
+        self.lock.release()
+
+        self.session_manager.remove_session(session)
+
+    def _session_name(self, peer, name):
```

Comment:
Maybe `_session_unique_id`?

Reply:
I think `_session_id` like in your other comment makes sense